src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
```

Comment:
Due to the fact that HTML is badly structured, you also need to change the CSS code after HTML changes.

Reply:
Not sure what exacly you meant here, but yeah, it's structured badly. I'll get better in no time ;)

Reply:
Michał meant that you need to change some selectors, because you have made changes to HTML.